test/core/MakerOperations.t.sol

```diff
@@ -1066,4 +1066,16 @@ contract MakerOperationsTest is MangroveTest, IMaker {
     vm.expectRevert("mgv/updateOffer/unauthorized");
     mgv.updateOfferByTick(olKey, Tick.wrap(0), 1 ether, 100_000, 30, ofrId);
   }
+
+  function test_new_offer_extremas() public {
+    mgv.setDensity96X32(olKey, 0);
+    vm.expectRevert("mgv/writeOffer/gives/tooBig");
+    uint ofr = mkr.newOfferByTick(Tick.wrap(0), MAX_SAFE_VOLUME + 1, 100_000, 0);
+    vm.expectRevert("mgv/writeOffer/gives/tooBig");
+    mkr.updateOfferByTick(Tick.wrap(0), MAX_SAFE_VOLUME + 1, 100_000, 0, ofr);
+    vm.expectRevert("mgv/writeOffer/tick/outOfRange");
+    mkr.updateOfferByTick(Tick.wrap(MAX_TICK + 1), 1, 100_000, 0, ofr);
+    vm.expectRevert("mgv/writeOffer/tick/outOfRange");
+    mkr.newOfferByTick(Tick.wrap(MAX_TICK + 1), 1, 100_000, 0);
```

Comment:
Should this also check that all of these succeed if `+ 1` is removed?

Reply:
yes, will add them